terraform/projects/app-ecs-services/alertmanager-service.tf

```diff
@@ -120,13 +128,38 @@ data "pass_password" "pagerduty_service_key" {
 data "template_file" "alertmanager_config_file" {
   template = "${file("templates/alertmanager.tpl")}"
 
+  # Even though we do not wish to send emails from staging, we can still include SMTP set up
```

Comment:
why don't we want to send emails from staging? can you explain?

Reply:
I think the wording needs to be improved, as we probably only want to target a verified email and don't expect the quota to go above the daily limit of `100`, whereas in production it might. I've already opened up a case to enable production access, it might take a few days for AWS to resolve.